File: python_gui_tkinter/KALU/GARBAGE1/ALLOUTDATED/1.py
import os, MaxPlus 
from PySide import QtCore, QtGui

fname = os.path.join(os.path.abspath(os.path.curdir), "my_ui.ui")
ui_type, base_type = MaxPlus.LoadUiType(fname)
print (ui_type)
print (base_type)

def buttonClicked():
    print ("pushButton clicked")

class MyWidget(base_type, ui_type):
  def __init__(self, parent = None):
    base_type.__init__(self)
    ui_type.__init__(self)
    self.setupUi(self)
    MaxPlus.AttachQWidgetToMax(self)
    pb = self.pushButton
    pb.clicked.connect(buttonClicked)

form = MyWidget()
form.show()

File: c++_backup/Amlyan/bala/1.cpp
#include<iostream>
using namespace std;

class a_
{
	public:
	int a,b;
	public:
		a_()
		{
			a=1;
			b=2;
		}
		void show()
		{
			cout<<"a = "<<a<<"b = "<<b<<endl;
		}
	
};


int main()
{	
	a_ a1,b1,c1;
	//a b1;
	//a c1;
	a1.show();
	b1.show();
	//c1 = a1+b1;
	c1.a = a1.a+b1.a;
	c1.b = a1.b+b1.b;
	c1.show();	
	return 0;
}



